lib/trento_web/plugs/app_jwt_auth_plug.ex

```diff
@@ -25,12 +25,21 @@ defmodule TrentoWeb.Plugs.AppJWTAuthPlug do
   def fetch(conn, _config) do
     with {:ok, jwt_token} <- read_token(conn),
          {:ok, claims} <- validate_access_token(jwt_token) do
+      sub = claims["sub"]
+      abilities = claims["abilities"]
+
       conn =
         conn
         |> Conn.put_private(:api_access_token, jwt_token)
-        |> Conn.put_private(:user_id, claims["sub"])
-
-      {conn, %{"access_token" => jwt_token, "user_id" => claims["sub"]}}
+        |> Conn.put_private(:user_id, sub)
+        |> Conn.put_private(:abilities, abilities)
```

Comment:
You can remove that from private, we add user_id for pow functioning, the abilities checking works without using the abilities from the private part of the conn, we can remove that field injecting

Reply:
Here we go https://github.com/trento-project/web/pull/3240/commits/a49e04ebca96631dce59f41c894bff017630fe85